api_core/google/api_core/grpc_helpers.py

```diff
@@ -149,6 +154,52 @@ def wrap_errors(callable_):
         return _wrap_unary_errors(callable_)
 
 
+def _create_secure_channel(
```

Comment:
probably makes sense to just roll the logic in this method into the `create_channel` method. I don't see a good reason for keeping them separate.

Reply:
_create_secure_channel has an additional arg "ssl_credential=None", do we want to add this to create_channel? Or if it's never a use case for user to pass in a ssl_credential to create_channel(), I can ignore it, and create a new ssl_credential every time. What's your thought?

Reply:
add the argument